frontend/src/scenes/session-recordings/player/player-meta/PlayerMetaLinks.tsx

```diff
@@ -24,7 +24,7 @@ function PinToPlaylistButton(): JSX.Element {
     const { maybePersistRecording } = useActions(sessionRecordingPlayerLogic)
     const nodeLogic = useNotebookNode()
 
-    const tooltip = logicProps.pinned ? 'Unpin from this list' : 'Pin to this list'
+    const tooltip = logicProps.pinned ? 'Unpin from this saved filter' : 'Pin to this saved filter'
```

Comment:
yep, i think pinning has to be to a list or a collection
doesn't make sense to me to pin to a saved filter 🙈